pkg/controllers/backup/controller.go

```diff
@@ -370,3 +365,8 @@ func (h *handler) setReconcilingCondition(backup *v1.Backup, originalErr error)
 	}
 	return backup, originalErr
 }
+
+// backupIsSingularAndComplete checks if the backup is a one-time backup and has not been modified
+func backupIsSingularAndComplete(backup *v1.Backup) bool {
+	return backup.Status.BackupType == "One-time" && backup.Generation == backup.Status.ObservedGeneration
```

Comment:
I think let's also introduce some `const string` types for valid backup types in the crd struct package definition. As this issue is related to code cleanup